dartagnan/src/main/java/com/dat3m/dartagnan/wmm/analysis/EmptyRelationAnalysis.java

```diff
@@ -0,0 +1,82 @@
+package com.dat3m.dartagnan.wmm.analysis;
+
+import com.dat3m.dartagnan.program.analysis.Dependency;
+import com.dat3m.dartagnan.program.analysis.ExecutionAnalysis;
+import com.dat3m.dartagnan.program.analysis.alias.AliasAnalysis;
+import com.dat3m.dartagnan.program.event.Event;
+import com.dat3m.dartagnan.utils.dependable.DependencyGraph;
+import com.dat3m.dartagnan.verification.Context;
+import com.dat3m.dartagnan.verification.VerificationTask;
+import com.dat3m.dartagnan.wmm.*;
+import com.dat3m.dartagnan.wmm.utils.EventGraph;
+import org.sosy_lab.common.configuration.Configuration;
+import org.sosy_lab.common.configuration.InvalidConfigurationException;
+
+import java.util.*;
+
+import static com.dat3m.dartagnan.program.event.Tag.*;
+import static java.util.stream.Collectors.toSet;
+
+public class EmptyRelationAnalysis extends NativeRelationAnalysis {
```

Comment:
The "empty" in the name tell me nothing. We need a more meaningful name. What about "coarse"?